webdriver/tests/bidi/script/call_function/result_node.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 from webdriver.bidi.modules.script import ContextTarget
+from ... import recursive_compare1
```

Comment:
I guess this didn't work?

```suggestion
from ... import recursive_compare
```